### DISCUSSION TITLE
Adetailer?

### DISCUSSION THREAD

**Discussion by Sirmadsen** [mcmonkeyprojects/SwarmUI#140] [General]
Trying again...

Is there a extension or plans to add something like adetailer to SwarmUI?

**Comment by mcmonkey4eva** [comment]
SwarmUI (not "Stable" anymore), has the equivalent of ADetailer documented here: https://github.com/mcmonkeyprojects/SwarmUI/blob/master/docs/Features/Prompt%20Syntax.md#automatic-segmentation-and-refining
You can use yolo models to get exactly equivalent to adetailer behavior, or clipseg to take it beyond with dynamic custom masking

**Reply by Sirmadsen** [reply]
Thanks, will have a look at it.
